app/controllers/admin/products_controller.rb

```diff
@@ -52,19 +52,20 @@ def search_products(search)
     end
 
     def new
-      @product = Product.new(use_simple_inventory: true).decorate
+      @product = Product.new(use_simple_inventory: true)
```

Comment:
@thermistor the removal of `decorate` here broke adding a new product! When I try to add a new product I get `undefined method `has_custom_seller_info?' for #<Product:0x007f911757b318>` without the `decorate`. Could you add a spec or see if there is an existing spec showing this injection?